lib/ui/painting.dart

```diff
@@ -3746,6 +3746,96 @@ class ImageShader extends Shader {
   void _initWithImage(_Image image, int tmx, int tmy, int filterQualityIndex, Float64List matrix4) native 'ImageShader_initWithImage';
 }
 
+/// A shader (as used by [Paint.shader]) that runs provided SPIR-V code.
+///
+/// NOTE: This API is in beta and does not yet work on web.
+/// See https://github.com/flutter/flutter/projects/207 for roadmap.
+///
+/// When initializing or updating the [floatUniforms], the length of float
```

Comment:
`[floatUniforms]` doesn't seem to point anywhere; use \`\` for non-hyperlinks

Reply:
done